training/src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -1,10 +1,13 @@
 package wolox.training.repositories;
 
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
 import wolox.training.models.Book;
 
+@Component
 public interface BookRepository extends JpaRepository<Book, Long> {
 
-    Book findByAuthor(String author);
+    List<Book> findByAuthor(String author);
```

Comment:
Wait, you need to return just one book by author. Don't hate me, I swear I'm a nice person. Check [this](https://docs.spring.io/spring-data/data-jpa/docs/current/reference/html/#repositories.limit-query-result), it may help you.

Reply:
jajaja no problem 👍